apis/axios.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosError } from 'axios';
+import LocalStorage from '@/utils/localStorage';
+import { AxiosResponse } from 'axios';
+
+const BASE_URL = 'https://sp-taskify-api.vercel.app/1-4/';
```

Comment:
```suggestion
const BASE_URL = 'https://sp-taskify-api.vercel.app/3-4/';
```

저희 주소 이거 아닌가요?!
`파트 3`-`4팀` 이니까

Reply:
저희 앞에 숫자는 기수 아닌가요??

Reply:
보니까 기수 맞는것같아요!

Reply:
죄송합니다 하하